.ci/setup.sh

```diff
@@ -11,15 +11,12 @@ ARCH=$(uname -m)
 
 
 if [[ $OS_NAME == "macos" ]]; then
+    # Check https://github.com/actions/runner-images/tree/main/images/macos for available
+    # versions of Xcode
+    sudo xcode-select -s /Applications/Xcode_14.3.1.app/Contents/Developer || exit 1
```

Comment:
```suggestion
    sudo xcode-select -s /Applications/Xcode_14.3.app/Contents/Developer || exit 1
```

Looks like the path has `Xcode_14.3.app/` not `Xcode_14.3.1.app/`: https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md#xcode